R/batchAnnotation.R

```diff
@@ -23,9 +23,11 @@ addFileLink <- function(batchCodeList, recordedBy, experiment, lsTransaction,
     fileName <- basename(reportFilePath)
     
     experimentCodeName <- experiment$codeName
+    entityType <- "experiment"
+    folderPath <- entityFileStorePaths[entityType]
```

Comment:
entityFileStorePaths is basically an enum lookup for folder paths in the racas package.